src/test/java/org/minions/devfund/katerinaanzoleaga/battleship/ShipTest.java

```diff
@@ -0,0 +1,280 @@
+package org.minions.devfund.katerinaanzoleaga.battleship;
+
+import junitparams.Parameters;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+//import org.junit.experimental.categories.Category;
+import junitparams.JUnitParamsRunner;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Ship test class.
+ */
+@RunWith(JUnitParamsRunner.class)
+public class ShipTest {
+    private BattleShip battleShip = new BattleShip();
+    private BattleCruiser battleCruiser = new BattleCruiser();
+    private Cruiser cruiser = new Cruiser();
+    private LightCruiser lightCruiser = new LightCruiser();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "lightCruiser" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)